red4ext-sys/src/interop.rs

```diff
@@ -45,13 +45,13 @@ impl From<CName> for u64 {
 impl CName {
     #[inline]
     pub const fn new(str: &str) -> Self {
+        if const_str::equal!(str, "None") {
```

Comment:
I think you can do this without a dependency:
```
match str.as_bytes() {
  b"None" => ..
  _ =>
}
```

Reply:
Sweet ! 😃 Done in cc75355